datalad/interface/ls.py

```diff
@@ -415,15 +407,22 @@ def convert_field(self, value, conversion):
             else:
                 return u'-'
         elif conversion == 'X':  # colored bool
-            chr, col = (self.OK, self.GREEN) if value else (self.NOK, self.RED)
-            return u"%s%s%s" % (col, chr, self.RESET)
+            if value:
+                mark, col = self.OK, ansi_colors.GREEN
+            else:
+                mark, col = self.NOK, ansi_colors.RED
+            return ansi_colors.color_word(mark, col)
         elif conversion == 'N':  # colored Red - if None
             if value is None:
                 # return "%s✖%s" % (self.RED, self.RESET)
-                return u"%s%s%s" % (self.RED, self.NONE, self.RESET)
+                return ansi_colors.color_word(self.NONE, ansi_colors.RED)
             return value
         elif conversion in {'B', 'R', 'U'}:
-            return u"%s%s%s" % ({'B': self.BLUE, 'R': self.RED, 'U': self.DATASET}[conversion], value, self.RESET)
+            return ansi_colors.color_word(
+                value,
+                {'B': ansi_colors.BLUE,
+                 'R': ansi_colors.RED,
+                 'U': self.DATASET}[conversion])
```

Comment:
`self.DATASET` is equivalent to `ansi_colors.DATASET`, though given that the code is `'U'`, perhaps `ansi_colors.UNDERLINE` makes more sense than the indirection.

Reply:
> `self.DATASET` is equivalent to `ansi_colors.DATASET`,

Ah good catch.  I didn't even realize ansi_colors.DATASET existed (though it is already used elsewhere in ls.py).

> though given that the code is `'U'`, perhaps `ansi_colors.UNDERLINE` makes more sense than the indirection.

True, though I don't care enough one way or the other to change the original decision to use DATASET.